### DISCUSSION TITLE
Increase the width of only one chunk output (follow-up question)

### DISCUSSION THREAD

**Discussion by werkstattcodes** [quarto-dev/quarto-cli#7257] [Q&A]
This is a follow-up question to this [question](https://github.com/quarto-dev/quarto-cli/discussions/3934).

I would like to change the output width for only one output (a dataframe), not the entire quarto document. How can I do this in the code chunk options? I tried e.g`. #| options(width=200)` but I get the following error message: 
`ERROR: YAMLException: can not read a block mapping entry; a multiline key may not be an implicit `

`#| width: 200` doesn't cause an error, but I do not see any change in the output's width; it's still truncated/cut off on the right side.

Alternatively, I would be also happy to have a horizontal scroll bar (for the output); but so far I have seen only `code-overflow: scroll`

I am aware that I could feed the output, e.g. into `gtable `or `reactable`, but I am interested in a simpler, direct solution.

Many thanks!

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````
